lambda_leg:
  type: position_controllers/JointTrajectoryController
  joints:
    - L_hip_joint
    - L_hip2_joint
    - L_thigh_joint
    - L_calf_joint
    - L_toe_joint
    - R_hip_joint
    - R_hip2_joint
    - R_thigh_joint
    - R_calf_joint
    - R_toe_joint
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 10  
